apps/admin_app/assets/css/promotions.css

```diff
@@ -0,0 +1,169 @@
+.App {
```

Comment:
This file has inconsistent indentation. Please rectify the indentation and either have an eslint rule in place like airbnb rules or prettier to auto correct the indentations.

Reply:
Alright. I have installed eslint for the same. Thanks :)